src/BenchmarkDotNet/Portability/Cpu/ProcCpuInfoParser.cs

```diff
@@ -61,8 +61,14 @@ internal static CpuInfo ParseOutput([CanBeNull] string content)
         private static double ParseFrequencyFromBrandString(string brandString)
```

Comment:
This method should return `Frequency`. Also, it would be nice to make it internal and write a few unit tests.

Reply:
So should CpuInfo ctor has nominalFrequency, minFrequency and maxFrequency arguments of type Frequency?

Reply:
Yes. We should use `Frequency` everywhere when it's possible.